Index: metadata.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/metadata.xml b/metadata.xml
--- a/metadata.xml	(revision 7fb7df368a794b43262d7fd57151e35b8849e008)
+++ b/metadata.xml	(date 1659625092466)
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <metadata>
-    <name>coopHUD *WIP*</name>
-    <directory>coop_hud_2731267631</directory>
-    <id>2731267631</id>
+    <name>coopHUD *DEV*</name>
+    <directory>coop_hud</directory>
+    <id>0</id>
     <description>
         [h1]Coop HUD - proper HUD for coop Isaac[/h1]
         This mod renders active items/pocket/trinkets in proper position for coop game up to 4 players.
Index: coopHUD_globals.lua
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/coopHUD_globals.lua b/coopHUD_globals.lua
--- a/coopHUD_globals.lua	(revision 7fb7df368a794b43262d7fd57151e35b8849e008)
+++ b/coopHUD_globals.lua	(date 1659625053892)
@@ -1,4 +1,4 @@
-coopHUD.VERSION = '0.8.2-WIP'
+coopHUD.VERSION = '0.8.2-DEV'
 coopHUD.showExtraInfo = false
 coopHUD.HUD_table = {}
 coopHUD.signals = {
@@ -20,7 +20,7 @@
     show_player_names = true,
     extra_hud = true,
     extra_hud_hide_on_battle = true,
-    h_trigger_key = -1,
+    h_trigger_key = Keyboard.KEY_H,
     stats = {
         show = true,
         hide_in_battle = true,
Index: main.lua
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.lua b/main.lua
--- a/main.lua	(revision 7fb7df368a794b43262d7fd57151e35b8849e008)
+++ b/main.lua	(date 1659625053909)
@@ -8,4 +8,5 @@
 include('coopHUD_hud.lua')
 include("callbacks.main.lua")
 --
-print('CoopHUD v.' .. tostring(coopHUD.VERSION) .. ' successfully!')
\ No newline at end of file
+print('CoopHUD v.' .. tostring(coopHUD.VERSION) .. ' successfully!')
+include("test.lua")
\ No newline at end of file
Index: test.lua
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.lua b/test.lua
new file mode 100644
--- /dev/null	(date 1659617542003)
+++ b/test.lua	(date 1659617542003)
@@ -0,0 +1,15 @@
+function coopHUD.test()
+	if coopHUD.players[1] == nil then
+		coopHUD.on_start(_, true)
+	end
+	if Input.IsButtonTriggered(Keyboard.KEY_J, 0) then
+
+	end
+	if Input.IsButtonTriggered(Keyboard.KEY_K, 0) then
+
+	end
+	if Input.IsButtonTriggered(Keyboard.KEY_I, 0) then
+
+	end
+end
+coopHUD:AddCallback(ModCallbacks.MC_POST_RENDER, coopHUD.test)
\ No newline at end of file
